parsers/PH.py

```diff
@@ -681,25 +682,39 @@ def fetch_production(
         .pipe(filter_generation)
         .pipe(match_resources_to_modes, logger)
         .pipe(aggregate_per_datetime_mode)
-        .pipe(pivot_per_mode)
+        .pipe(add_production_mix_column)
     )
 
     production_breakdown = ProductionBreakdownList(logger)
-    for tstamp, row in df.iterrows():
+    df = df.set_index("datetime")
+    for dt in df.index.unique():
+        df_dt = df.loc[dt]
         production_mix = ProductionMix()
-        for mode in [m for m in row.index if "production." in m]:
-            production_mix.add_value(mode.replace("production.", ""), row[mode])
         storage_mix = StorageMix()
-        if "storage.hydro" in row.index:
-            storage_mix.add_value("hydro", row["storage.hydro"])
+        for idx, row in df_dt.iterrows():
+            if row["production_mix"] == "production":
+                production_mix.add_value(row["mode"], row["production"])
+            elif row["production_mix"] == "storage":
+                if row["mode"] == "hydro_storage":
+                    mode = "hydro"
+                else:
+                    mode = row["mode"]
+                storage_mix.add_value(mode, row["production"])
```

Comment:
```suggestion
                storage_mix.add_value(mode, -row["production"])
```
This gets us every time...
can we add something in the long term planning to fix this?